generators/client/prompts.js

```diff
@@ -188,3 +189,23 @@ function askForClientThemeVariant() {
         this.clientThemeVariant = this.jhipsterConfig.clientThemeVariant = prompt.clientThemeVariant;
     });
 }
+
+function askForAdminUi() {
+    if (this.existingProject) {
+        return undefined;
+    }
+
+    const skipClient = this.skipClient;
+
+    const PROMPT = {
```

Comment:
why is this uppercase?

Reply:
I copied this from another prompt in the client prompt file. If you want I cant rename all the PROMPT to prompt.

Reply:
Ok so I think this have been done to avoir misunderstanding between PROMPT and prompt in the callback ;)

Reply:
Really? Wow. Talk about dev laziness... I know naming things is one of the most difficult things we devs have to do but still...

Reply:
The PROMPT thing is a param object, its name could be "promptParams". The prompt function param is regular param, which represents a prompt result, an answer.